tests/www/test_views.py

```diff
@@ -62,7 +62,14 @@
 from airflow.utils.timezone import datetime
 from airflow.utils.types import DagRunType
 from airflow.www import app as application
-from airflow.www.views import ConnectionModelView, get_safe_url, truncate_task_duration
+from airflow.www.views import (
+    ConnectionModelView,
+    build_dag_sorting_query,
+    dag_query_for_key,
+    query_ordering_transform_for_key,
+    get_safe_url,
+    truncate_task_duration
```

Comment:
```suggestion
    get_safe_url,
    query_ordering_transform_for_key,
    truncate_task_duration
```

to fix the failing test